fr.md

```diff
@@ -231,4 +226,4 @@ Pour faire valoir une opposition ou une révocation, vous pouvez contacter notre
 
 ---
 
-_Date de la déclaration sur la protection des données : 08/05/2018_
```

Comment:
Do we want to automatize this? A minuscole and inobtrusive solution could be found